packages/cardpay-reward-programs/cardpay_reward_programs/main.py

```diff
@@ -49,11 +35,11 @@ def run_reward_program(
     """
     with open(AnyPath(parameters_file), "r") as stream:
         parameters = json.load(stream)
-    rule = to_rule(parameters)
-    start_block = rule.start_block
-    end_block = rule.end_block
-    results_df = rule.run(start_block, end_block)
-    payment_list = rule.df_to_payment_list(results_df, payment_cycle, reward_program_id)
+    
+    for subclass in Rule.__subclasses__():
```

Comment:
A safer approach than globals, but achieves the same sort of thing. Looking for a class based on name, but it's limited to Rules.

Reply:
Strongly agree!